SKILL.tsv

```diff
@@ -5131,11 +5131,11 @@ SKILL_20150918_005130	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRati
 SKILL_20150918_005131	Carve Statue Duration: #{CaptionRatio}# seconds{nl}Consumes 1 Ash Wood 1
 SKILL_20150918_005132	{#339999}{ol}[Goddess' Blessing]{/}{/} Duration: #{CaptionRatio}# seconds{nl}Goddess Statue Duration: 30 seconds{nl}Consumes 1 Cedar Wood
 SKILL_20150918_005133	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio}#%{nl}Up to 5 users{nl}Maximum Duration: 10 seconds
-SKILL_20150918_005134	Pushes the enemies outside and creates the wall of protection which doesn't allow others to enter for some time. It will increase DEF of the allies inside the wall.
-SKILL_20150918_005135	Magic DEF #{CaptionRatio}# Increase{nl}Duration #{CaptionTime}#Secs
+SKILL_20150918_005134	Pushes enemies away and creates a barrier of protection which doesn't allow others to enter for some time. Magic defense of your allies inside the barrier will increase.
```

Comment:
Magic defense of your allies inside the barrier will increase. -> Increases magic defense of allies who are inside the barrier.